src/views/Settings.vue

```diff
@@ -33,7 +33,62 @@
         @input="setMinLowerChar"
       />
     </div>
+  <div class = "configurations">
+    <h1>Configurations</h1>
+      <ul>
+        <li><button class = "button1" v-on:click="
+        config.minLength = 6,
+        config.minSpecialChar = 0,
```

Comment:
State should use mutations, defined in `src/store/models/configuration.js`
See the [Mutations docs](https://vuex.vuejs.org/guide/mutations.html) for about why

Can call the methods defined in the `methods` object in the script section in this file